exchanges/okcoin/okcoin_wrapper.go

```diff
@@ -310,47 +310,47 @@ func (o *OKCoin) FetchTradablePairs(ctx context.Context, a asset.Item) (currency
 	}
 	pairs := make([]currency.Pair, len(prods))
 	for x := range prods {
+		if prods[x].State != "live" {
+			continue
```

Comment:
Continuing here will leave holes in the above allocation, so please change `pairs := make([]currency.Pair, len(prods))` to `pairs := make([]currency.Pair, 0,  len(prods))` then append.